schema/graph.json

```diff
@@ -155,7 +219,22 @@
           "data": {},
           "metadata": {
             "type": "object",
-            "additionalProperties": true
+            "additionalProperties": true,
+            "properties": {
+              "route": {
+                "type": "integer",
+                "description": "Route identifier of a graph edge"
```

Comment:
What is this identifier used for? Why is it a number? What does the number refer?

Reply:
The route number is used for edge color